public/documentation/getting_started/remote_debugging.html

```diff
@@ -0,0 +1,1206 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><style>body {
```

Comment:
It looks like this file was included on accident.  I removed it.  The "html" files all go in the base "www" directory.  They are auto-generated from the "public" .md files.